Makefile

```
@@ -1,18 +1,13 @@
-.PHONY: clean
-clean:
-	go run github.com/google/ko@latest delete -f config/
+# Those are callable targets
+TASKS = build clean deploy undeploy update update-deps update-codegen \
+  test e2e unit built-test
 
-.PHONY: deploy
-deploy:
-	go run github.com/google/ko@latest apply -f config/
+all: build
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)